app/views/idv/doc_auth/_ssn_init.html.erb

```diff
@@ -16,6 +16,16 @@
   <%= new_window_link_to(t('doc_auth.instructions.learn_more'), MarketingSite.security_and_privacy_practices_url) %>
 </p>
 
+<% if IdentityConfig.store.proofer_mock_fallback %>
+  <div class="usa-alert usa-alert--info margin-bottom-4" role="status">
+    <div class="usa-alert__body">
+      <p class="usa-alert__text">
+        <%= t('doc_auth.instructions.test_ssn') %>
+      </p>
+    </div>
+  </div>
+<% end %>
+
```

Comment:
Minor post-merge observation: We could use `AlertComponent` to simplify this, as above:

```suggestion
<% if IdentityConfig.store.proofer_mock_fallback %>
  <%= render AlertComponent.new(class: 'margin-bottom-4') { t('doc_auth.instructions.test_ssn') } %>
<% end %>

```